dal/dal.py

```diff
@@ -0,0 +1,94 @@
+from mongodb.mongodb_handler import (
+    update_item_status,
+    update_patron_items,
+    insert_document,
+    delete_document, delete_bill, update_bill, insert_bill
+)
+
+
+def update_item_dal(isbn, is_borrowed):
```

Comment:
Also, what are these function names???? 
For example, rename this to `update_item` so when you call it, it wil be `dal.update_item(...)`